service/pom.xml

```diff
@@ -154,7 +124,32 @@
 			<artifactId>commons-io</artifactId>
 			<version>2.11.0</version>
 		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>com.h2database</groupId>
+			<artifactId>h2</artifactId>
+			<scope>test</scope>
```

Comment:
Bitte noch den Scope entfernen, da es sonst nicht in Eclipse startet.

Reply:
IMHO sollte nicht ein H2 Datenbank für die Implementierung konfiguriert werden, die nur in Tests Anwendung findet, sondern `Failed to configure a DataSource: 'url' attribute is not specified and no embedded datasource could be configured.` sagt aus, dass aus Sicht von Spring keine `datasource` konfiguriert wurde. Da Ihr Euch erst zur Laufzeit für den SQL-Server oder Postgres entscheidet, sollte Spring das beim Start mit `@SpringBootApplication(exclude = {DataSourceAutoConfiguration.class })` ignorieren.